datastore/datastore-io/src/googMain/kotlin/androidx/datastore/io/IO.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.datastore.io
+
+import java.io.File
+import java.io.InputStream
+import java.io.OutputStream
+
+actual class DatastoreOutput(val outputStream:OutputStream) {
```

Comment:
i thought we were going to typealias these to InputStream / BufferedSink.

Would be good to see how you'll write 1 Serializer in our codebase that compiles with both of these, it feels impossible to write that in common right now unless i'm missing something. The DatastoreOutput expect has no methods so how would you use it from common? (feels like option 3)